bukuserver/README.md

```diff
@@ -20,33 +20,28 @@ So be sure to have `python3`, `python3-pip` , `python3-dev`, `libffi-dev` packag
 #### Dependencies
 
 ```
-$ python3 -m pip install --user --upgrade pip
```

Comment:
Depending on OS, `pip install --user` is no longer allowed (unless you pass `--break-system-packages` as well)